src/components/Posts/index.js

```diff
@@ -294,8 +294,10 @@ function Posts(props) {
     }, [postCreatedAt]);
 
     useEffect(() => {
-        const localLikeCount = getLocalLikeCountForPost(props.note.id);
-        setVotesCount(props.note.voteCount + localLikeCount);
+        // const localLikeCount = getLocalLikeCountForPost(props.note.id);
+        // console.log(localLikeCount);
+        console.log(props.note.voteCount);
```

Comment:
??

Reply:
Removed the log statements and commited the changes please check